plugins/midas theme/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Please change the folder name to not include a space: "plugins/midas theme/" > "plugins/MidasTheme/"